client/src/Features/Annotations/Annotations.tsx

```diff
@@ -0,0 +1,249 @@
+import DataTable from '@/Components/DataTable/DataTable';
+import React, { useCallback, useEffect, useState } from 'react';
+import { ArrowRightIcon } from '@heroicons/react/24/outline';
+import {
+  calculateDate,
+  calculateSampleCount,
+  getOriginalFrequency,
+  getFrequency,
+  getSeconds,
+  validateFrequency,
+  validateDate,
+} from '@/Utils/rfFunctions';
+import AutoSizeInput from '@/Components/AutoSizeInput/AutoSizeInput';
+
+export const Annotations = ({ meta, totalIQSamples, updateSpectrogram }) => {
+  const [metadata, setMetadata] = useState(meta);
+  const [parents, setParents] = useState([]);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
```

Comment:
is this actually needed because you aready initialize parents ad an empty array.,

Reply:
No! When I was having issues with my tests and added it in as a test and forgot to remove, thanks :D